luadox/render/base.py

```diff
@@ -32,15 +32,15 @@ def __init__(self, parser: Parser):
         self.ctx = parser.ctx
 
     def copy_file_from_config(self, section: str, option: str, outdir: str) -> None:
-        fname = self.config.get(section, option, fallback=None)
-        if not fname:
+        fnames = self.config.get(section, option, fallback='').split()
+        if not fnames:
             return
```

Comment:
No need for this anymore.  If the list is empty then the iteration that follows will be a no-op.